github_rules/github_branch_policy_override.yml

```diff
@@ -0,0 +1,27 @@
+AnalysisType: rule
+Filename: github_branch_policy_override.py
+RuleID: GitHub.Branch.PolicyOverride
+DisplayName: GitHub Branch Protection Policy Override
+Enabled: false
+LogTypes:
+  - GitHub.Audit
+Tags:
+  - GitHub
+Severity: High
+Description: Bypassing branch protection controls could indicate malicious use of admin credentials in an attempt to hide activity.
+Runbook: Verify that the GitHub admin performed this activity and validate its use.
+SummaryAttributes:
+  - p_any_ip_addresses
```

Comment:
I was under the impression from your comments on slack that these logs did not typically include IP addresses? Are we expecting this field to be populated?

Reply:
Whoops! You are right - this is copy pasta.  I'll remove/replace with more appropriate ones throughout.